pkg/cmd/issue/view/view.go

```diff
@@ -65,6 +69,8 @@ func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Comman
 	}
 
 	cmd.Flags().BoolVarP(&opts.WebMode, "web", "w", false, "Open an issue in the browser")
+	cmd.Flags().IntVarP(&opts.Comments, "comments", "c", 1, "View issue comments sorted by most recent")
```

Comment:
Is this description descriptive enough to convey that issues will be retrieved by most recent first? 
cc @ampinsk

Reply:
Hm this would make me think that they were displayed most recent at the top?

Reply:
Yes, now that you say that I agree.

Reply:
Do we have to mention sorting at all? If you said nothing I'd just assume chronologically anyway.